apache/apache.go

```diff
@@ -94,21 +76,33 @@ func getMetrics(webserver string, metrics []string) ([]plugin.MetricType, error)
 			if err != nil {
```

Comment:
Since the two endpoints have different metrics available, how should this be handled? Should apache version be added to the config so that it advertises different metrics based on what version is specified or should it advertise everything and then return only what it collects?